openqa-investigate

```diff
@@ -137,7 +137,13 @@ investigate() {
         [[ ! "$old_name" =~ investigate:retry$ ]] && echo "Job is ':investigate:' already, skipping investigation" && return 0
         investigate_origin="$(echo "$job_data" | runjq -r '.job.settings.OPENQA_INVESTIGATE_ORIGIN')" || return 1
         origin_job_id=${investigate_origin#$host_url/t}
-        "${client_call[@]}" -X POST jobs/"$origin_job_id"/comments text="Investigate retry job: $host_url/t$id failed, likely not a sporadic test"
+        out=$("${client_call[@]}" -X POST jobs/"$origin_job_id"/comments text="Investigate retry job: $host_url/t$id failed, likely not a sporadic test" 2>/dev/null)
+        # meanwhile the original job might have been deleted already, handle
+        # gracefully
+        if [[ $(echo "$out" | runjq .error_status) != "404" ]]; then
```

Comment:
If it actually worked the error status is not 404 as well but we likely shouldn't run this block.

Reply:
huh?

Reply:
oh, I get it now. Something is missing …

Reply:
updated to check the exit code first